sample-code/src/langchain-azure-openai.ts

```diff
@@ -95,7 +95,7 @@ export async function invokeRagChain(): Promise<string> {
       Use the following pieces of retrieved context to answer the question.
       If you don't know the answer, just say that you don't know.
       Use ten sentences maximum and keep the answer concise.
-      Also prioritize code examples.
+      Also prioritize code examples, e.g., showing the class instantiation.
```

Comment:
[q] What about: Always include code snippets in your answer including class instantiation.

Reply:
Done